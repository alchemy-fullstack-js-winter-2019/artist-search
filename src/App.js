import React from 'react';
import {
  BrowserRouter as Router,
  Route,
  // Redirect,
  // Link 
} from 'react-router-dom';
import Search from '../src/components/Search';
import Songs from '../src/components/songs/Songs';
import Song from '../src/components/songs/Song';

function App() {
  return (
    <Router>
        < >
          <Route exact path='/' component={Search} />
          {/* <Route exact path='/about' component={About} /> */}
          <Route exact path='/artist/:id' component={Songs} />
          <Route exact path='/work/:artist/:song' component={Song} />
          {/* <Redirect to='/home' /> */}
        </> 
    </Router>
  );
}

export default App;
