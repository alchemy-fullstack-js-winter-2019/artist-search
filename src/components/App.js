import React, { Component } from 'react';
import {
    BrowserRouter as Router,
    Route,
    Switch
} from 'react-router-dom';
import Search from './search/Search';
import Artist from './artist/Artist';


export default class App extends Component {
   

    render() {
        return (
            <Router>
                <Switch>
                    <Route exact path="/" component={Search}/>
                    <Route exact path='/artist/:id' component={Artist}/>
                </Switch>
            </Router>
        );
    }
}






