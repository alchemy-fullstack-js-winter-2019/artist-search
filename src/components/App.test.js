import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('App', () => {
  it('is an App snapshot', () => {
    const tree = renderer.create(
      <App />
    ).toJSON(); 
    expect(tree).toMatchSnapshot();
  });
});



