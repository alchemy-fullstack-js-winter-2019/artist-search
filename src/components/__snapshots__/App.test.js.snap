// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches a snapshot 1`] = `
Array [
  <h1>
    Sup Searchie
  </h1>,
  <form
    onSubmit={[Function]}
  >
    <input
      name="query"
      onChange={[Function]}
      type="text"
    />
    <button>
      Yo
    </button>
  </form>,
  <ul />,
]
`;
